lms/djangoapps/edraak_university/views.py

```diff
@@ -116,14 +117,16 @@ def get_context_data(self, **kwargs):
         data = super(UniversityIDView, self).get_context_data(**kwargs)
         mktg_enabled = settings.FEATURES.get('ENABLE_MKTG_SITE')
         university_settings = self.get_university_settings()
+        registration_end_date = university_settings.registration_end_date if university_settings else None
+        formatted_registration_end_date = strftime_localized( registration_end_date, format="SHORT_DATE") if registration_end_date else None
 
```

Comment:
~~You can squash all that into a single line below like it used to be, you are only adding the strftime_localized function~~